### PR TITLE
Update dry-configurable, use yt-dlp by default

### PR DESCRIPTION
**Pull request by ivan-leschinsky**

### CODE REVIEW

**Comment by ivan-leschinsky** [comment]
@maxhollmann Can you please review it? 😄

**Comment by maxhollmann** [comment]
Looks good, but would you change the `setting :executable, default: 'youtube-dl'` to `yt-dlp`?

And thank you for the contribution :)

File: README.md [hunk 1]
```diff
@@ -42,6 +44,12 @@ state = YoutubeDL.download('https://www.youtube.com/watch?v=MmIWve5bUpU', format
   .call
 ```
 
+Configurable with:
+```ruby
+YoutubeDL::Command.config.executable = 'yt-dlp' # if you're using yt-dlp
+YoutubeDL::Command.config.default_options = { some_option: true }
+```
```
Comment (maxhollmann) [inline README.md]: I'd say put this in a new section, `### Configuration` above Events.

Comment (maxhollmann) [reply README.md]: Oh, and change it to `'youtube-dl'`, since yt-dlp will be the default.

**Review by maxhollmann — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ivan-leschinsky) [reply README.md]: I've changed it initially to `yt-dlp`, but then changed back as it is default for your repo 😀

**Comment by ivan-leschinsky** [comment]
@maxhollmann Updated. I also removed `version.rb` change as it is probably better to do it after merge and as it is major change you'll update it to correct version by yourself.

**Review by maxhollmann — APPROVED** [review]

**Comment by maxhollmann** [comment]
Released as 0.3.0 :)
Thanks again for your help!

**Pull request merged**